node-backend/constant/responseConstants.js

```diff
@@ -73,3 +73,10 @@ module.exports.INVALID_BADGES_MESSAGE = "Invalid badges data";
 module.exports.INVALID_TOKEN_CODE = "invalid_token";
 
 module.exports.INVALID_TOKEN_MESSAGE = "Invalid token data";
+
+module.exports.NOT_REPEATED_HI5_CODE = "not-give-repeat-hi5";
```

Comment:
constant name should  NO_REPEAT_HI5_CODE
                                     NO_REPEAT_HI5_MESSAGE

not-give-repeat-hi5  ---> duplicate-hi5-not-allowed